Source/Tests/Tests.Generator/Dialog/TestAssets/BlankProject.lvproj

```diff
@@ -0,0 +1,16 @@
+﻿<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
Is another option just to have the test script a blank project in a temp directory? What are the advantages of doing it this way?

Reply:
You could script it -- and for this particular test that shouldn't be a problem...  But I can anticipate that we might want to execute other tests against a blank project that has disk presence, due to the generator's requirement that it's running in the context of a project that has been saved.  So it just struck me as simplest to put a blank project on-disk as a test asset for all to share.

Reply:
Ok. I'm fine with that if we think it has usefulness in the future.